src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/combination.html.twig

```diff
@@ -23,17 +23,3 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  *#}
 {% extends '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit_base.html.twig' %}
```

Comment:
What is the interest of this file, now ?

Reply:
I think @jolelievre idea was to still have theme files even they are empty, but maybe it was only mistake, could I delete empty theme files @jolelievre ?

Reply:
related comment 
https://github.com/PrestaShop/PrestaShop/pull/27216/files#r845959165
Though I would say we should remove whats unused stuff. Its not that hard to create empty twig file again later if we need it (and it is very easy to forget deleting it :D)

Reply:
Yeah you can delete empty form theme Since bulk form a combination form are gonna share the same I guess only one is needed so you could remove this one https://github.com/PrestaShop/PrestaShop/blob/1dc763baf576c48318c2d86f7813a1d17e797d9e/src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/FormTheme/bulk_combination.html.twig

We can always re-introduce it if we someday need a dedicated theme for bulk

Reply:
However, you shouldn't remove this one yet.
It's great now thanks to the `CombinationImagesChoiceType` we have a cleaner widget override via the `combination_images_choice_widget`

However, you put it in the `prestashop_ui_kit_base.html.twig` file but it should still be here. The Prestashop file is meant for generic components that are cross domains. But since this one is specific to combination it's better to keep it apart in this dedicated form theme. So you should move back the template here.